Rakefile

```diff
@@ -197,7 +198,7 @@ task :spec => ["spec:init"] do
     BUNDLE_GEMFILE='#{PUPPET_SRC}/Gemfile' \
     GEM_HOME='#{TEST_GEMS_DIR}' GEM_PATH='#{TEST_GEMS_DIR}' \
     lein #{profile} run -m org.jruby.Main \
-      -I'#{PUPPET_LIB}' -I'#{PUPPET_SPEC}' -I'#{FACTER_LIB}' -I'#{PUPPET_SERVER_RUBY_SRC}' \
```

Comment:
I think we can remove the `PUPPET_SPEC` definition above with this change.

Reply:
done